src/main.cpp

```diff
@@ -581,6 +474,126 @@ MainFrame::MainFrame(wxWindow *parent) : TopFrame(parent)
 
     m_ckboxSNR->SetValue(wxGetApp().m_snrSlow);
     setsnrBeta(wxGetApp().m_snrSlow);
+}
+
+//-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=--=-=-=-=
+// Class MainFrame(wxFrame* pa->ent) : TopFrame(parent)
+//-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=--=-=-=-=
+MainFrame::MainFrame(wxWindow *parent) : TopFrame(parent)
+{
+    m_zoom              = 1.;
+
+    #ifdef __WXMSW__
+    g_logfile = stderr;
+    #else
+    g_logfile = stderr;
+    #endif
+
+    // Init Hamlib library, but we dont start talking to any rigs yet
+
+    wxGetApp().m_hamlib = new Hamlib();
+
+    // Init Serialport library, but as for Hamlib we dont start talking to any rigs yet
+
+    wxGetApp().m_serialport = new Serialport();
+    wxGetApp().m_pttInSerialPort = new Serialport();
+    
+    // Check for AVX support in the processor.  If it's not present, 2020 won't be processed
+    // fast enough
+    checkAvxSupport();
+    if(!isAvxPresent)
+        m_rb2020->Disable();
+
+    tools->AppendSeparator();
+    wxMenuItem* m_menuItemToolsConfigDelete;
+    m_menuItemToolsConfigDelete = new wxMenuItem(tools, wxID_ANY, wxString(_("&Restore defaults")) , wxT("Delete config file/keys and restore defaults"), wxITEM_NORMAL);
+    this->Connect(m_menuItemToolsConfigDelete->GetId(), wxEVT_COMMAND_MENU_SELECTED, wxCommandEventHandler(MainFrame::OnDeleteConfig));
+
+    tools->Append(m_menuItemToolsConfigDelete);
+
+    loadConfiguration_();
+    
+    if(wxGetApp().m_show_wf)
```

Comment:
@drowe67, we don't currently have these as editable within the application. Should these options just go away, or is there value in still letting people manually edit their config files to disable these tabs?

Reply:
Good point - yes it's fine to remove these options.  I think they are there for historical reasons - GUI code that was written by another author very early in the project.

Reply:
Sounds good. Removed. 👍